src/crosshook-native/src/styles/palette.css

```
@@ -0,0 +1,221 @@
+  background: linear-gradient(135deg, rgba(74, 125, 181, 0.24), rgba(107, 163, 217, 0.12));
+  color: var(--crosshook-color-text);
+  box-shadow: 0 14px 24px rgba(74, 125, 181, 0.2);
+}
+
+.crosshook-palette__row-icon {
+  width: 1.15rem;
+  height: 1.15rem;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)